doc/source/examples/similarity_explanations_20ng.ipynb

```diff
@@ -0,0 +1,1022 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    data = fetch_20newsgroups(shuffle=True, random_state=1, subset='all',
</pre><p>The dataset is quite large if you call the method with <code>subset='all'</code> . It froze my personal computer with 16GB or RAM. Consider using only the <code>train/test</code> subset which should fit in around 7-8 GB </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/650/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>In addition to this, please add a warning before this cell with the expected memory consumption (orange box, see https://nbsphinx.readthedocs.io/en/0.8.8/markdown-cells.html?highlight=warning#Info/Warning-Boxes).</p>